KK_ButtPlugin/Hooks.cs

```diff
@@ -13,8 +13,8 @@ public static void InstallHooks()
         private static class HSceneTriggers
         {
             [HarmonyPostfix]
-            [HarmonyPatch(typeof(HFlag), nameof(HFlag.Start))]
-            public static void Start(HFlag __instance)
+            [HarmonyPatch(typeof(HSceneProc), nameof(HSceneProc.Start))]

```

Comment:
Sadly, this suffers from the same problem as GameCustomFunctionController, which is that H scenes in VR have nothing to do with HSceneProc. They also have VRHandCtrl instead of HandCtrl, both derived directly from MonoBehaviour. If you want portable touch detection, you should probably look for the effects it has either on the HFlag or on the female's ChaControl.

Reply:
I say, let's forget about syncing hand movements for now, and let's do it in another PR. It wasn't part of the issue anyway.

Reply:
For the record, the correct way would be using HFlag.xy, but please don't add it now. It's too much for one PR. I'll deal with it after next release.

Reply:
code removed from this PR in favor of opening another